tests/vmi_configuration_test.go

```diff
@@ -2149,22 +2149,33 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 		})
 
 		It("[test_id:1681]should set appropriate cache modes", func() {
-			vmi := tests.NewRandomVMI()
+			tmpHostDiskDir := tests.RandTmpDir()
+			memory := "128Mi"
```

Comment:
Seems redundant, do we need the guest to boot?

Reply:
Maybe you are right! I was just trying to leave things as it was.
But this could save time and cognitive resources.